src/ralph/conf.py

```diff
@@ -145,6 +152,39 @@ class DatabaseBackendSettings(BaseModel):
     CLICKHOUSE: ClickhouseDatabaseBackendSettings = ClickhouseDatabaseBackendSettings()
 
 
+# Active HTTP backend Settings.
+
+
+class HeaderParameters(BaseModel):
+    """Pydantic model for additionnal client options."""
+
+    class Config:  # pylint: disable=missing-class-docstring # noqa: D106
+        extra = Extra.allow
+
+
+class LRSHeaders(HeaderParameters):
+    """Pydantic model for LRS headers."""
+
+    X_EXPERIENCE_API_VERSION: str = "1.0.0"
```

Comment:
Why not the last version as default?
```suggestion
    X_EXPERIENCE_API_VERSION: str = "1.0.3"
```